tests/v2/validation/provisioning/README.md

```diff
@@ -137,7 +137,18 @@ provisioningInput is needed to the run the RKE2 tests, specifically kubernetesVe
 
```

Comment:
I think this statement in the readme file will have to be changed? `nodes are divided by "|"` ?

Reply:
After an offline discussion - Can we change it to something like `“{nodeRoles : True/false, Quantity: <qty>},”`